README.md

```diff
@@ -619,6 +621,131 @@ When enabled, the overall patient-level results will contain the raw results of
 ];
 ```
 
+## Composite Measures
+
+:warning: Composite measure support is highly experimental, and may change as the [specification](https://build.fhir.org/ig/HL7/cqf-measures/composite-measures.html) evolves :warning:
+
+`fqm-execution` supports the calculation of [composite measures](https://build.fhir.org/ig/HL7/cqf-measures/composite-measures.html) under the following conditions:
+
+- The provided measure bundle **must** contain only one composite `Measure` resource (i.e. a FHIR `Measure` resource with `composite` as the scoring code)
+  - See the [conformance requirements of a composite measure](https://build.fhir.org/ig/HL7/cqf-measures/composite-measures.html#component-quality-measures) and the [CQFMCompositeMeasure profile](https://build.fhir.org/ig/HL7/cqf-measures/StructureDefinition-composite-measure-cqfm.html) for specific guidance regarding a composite `Measure` resource
+- The provided measure bundle **must** contain all component `Measure` resources and their dependent `Library` resources needed for calculation
```

Comment:
If I remove the FHIRHelpers library from the composite measure bundle in the `test/integration` folder, the output is the same as when the FHIRHelpers library was included. Is this expected?

Reply:
Probably, because in those libraries as written, FHIRHelpers is never actually used in the CQL AFAIK. So no statements will throw an error since that library is never actually needed. It probably shouldn't even be included on that composite bundle in the first place